Firestore/core/src/firebase/firestore/remote/grpc_root_certificate_finder_apple.mm

```diff
@@ -16,14 +16,17 @@
 
 #include "Firestore/core/src/firebase/firestore/remote/grpc_root_certificate_finder.h"
 
+#import <objc/runtime.h>
+
 #include <string>
 
+#import "Firestore/Source/Core/FSTFirestoreClient.h"
```

Comment:
Optional: I'm not sure this `import` is necessary, maybe try just deleting it?

Reply:
Done. I think we had this before in order to find a class to load the FirebaseFirestore framework's bundle, but doing this reflectively seems cleaner from a layering perspective.